source/wp-content/themes/wporg-news-2021/block-template-parts/footer-archive.html

```diff
@@ -3,7 +3,7 @@
 	<!-- wp:group {"className":"follow-the-code"} -->
 	<div class="wp-block-group follow-the-code">
 		<!-- wp:heading {"level":4} -->
-		<h4>Follow The Code</h4>
+		<h5>Follow The Code</h5>
```

Comment:
The attribute here should also be updated, otherwise you'll get a broken block in the editor - the `level` is used to determine which `h*` to render.